scrunch/datasets.py

```diff
@@ -1628,6 +1632,49 @@ def _reload_variables(self):
             self._vars = self._catalog.index.items()
 
 
+class MissingRules(dict):
+    """
+    Handles variables missing rules in a dict fashion.
+    del var['skipped']  --> deletes a missing rule
```

Comment:
I believe this should be `del var.missing_rules['skipped']  --> deletes a missing rule`